src/pages/cloud-native-management/gitops/performance-management.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import SEO from "../../../components/seo";
+import PerformanceManagementPage from "../../../sections/gitops/PerformanceManagementPage";
+
+const Page = () => {
+  return (
+    <>
+      <PerformanceManagementPage />
+    </>
+  );
+};
+export default Page;
+export const Head = () => {
+  // TODO Change this
```

Comment:
@Chadha93 what to add in title and description?

Reply:
Uh-oh. I never delivered in this to @Udit-takkar. Lee has an outstanding action item here.